ui/src/app/components/genomics-extraction-table.tsx

```diff
@@ -230,17 +237,19 @@ const FailedRequestMessage = () => <div style={{textAlign: 'center'}}>
   </FlexRow>
 </div>;
 
-export const GenomicsExtractionTable = fp.flow(withCurrentWorkspace())(({workspace}) => {
-  const [extractionJobs, setExtractionJobs] = useState(null);
+export const GenomicsExtractionTable = fp.flow(
+  withCurrentWorkspace(),
+  withStore(genomicExtractionStore, 'genomicExtraction')
+)(({workspace, genomicExtraction}) => {
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     dataSetApi().getGenomicExtractionJobs(workspace.namespace, workspace.id)
-      .then(resp => setExtractionJobs(resp.jobs))
+      .then(resp => updateGenomicExtractionStore(workspace.namespace, resp.jobs))
       .finally(() => setIsLoading(false));
   }, [workspace]);
 
-  const requestFailed = !isLoading && extractionJobs === null;
+  const requestFailed = !isLoading && genomicExtractionStore === null;
```

Comment:
is the `genomicExtractionStore` we want to compare here or the `.get()` call to it?

Reply:
if so, maybe just add a line like `const extractionJobs = genomicExtraction[workspace.namespace]` right above this?

If so, we should be able to keep the rest of the code below the same.

Reply:
ope definitely the `.get()`